keras_retinanet/bin/train.py

```diff
@@ -113,6 +113,20 @@ def create_callbacks(model, training_model, prediction_model, validation_generat
     lr_scheduler = keras.callbacks.ReduceLROnPlateau(monitor='loss', factor=0.1, patience=2, verbose=1, mode='auto', epsilon=0.0001, cooldown=0, min_lr=0)
     callbacks.append(lr_scheduler)
 
+    if args.log_dir:
+        tb = keras.callbacks.TensorBoard(
+            log_dir                 =   args.log_dir,
+            histogram_freq          =   0,
+            batch_size              =   args.batch_size,
+            write_graph             =   True,
+            write_grads             =   False,
+            write_images            =   False,
+            embeddings_freq         =   0,
+            embeddings_layer_names  =   None,
+            embeddings_metadata     =   None
```

Comment:
The PEP8 check is unhappy about the multiple spaces after the operator. The aligning is fine (please keep it :) ), but it wants only a single space after the `=`

Reply:
Yes PEP8 hates me :) Do you want 1 space after `=`, or do you want fully PEP8 compliance with regards to the space.
E.g `write_images            = False,` or `write_images            =False,`?

Reply:
Definitely one space, PEP8 can go cry in a corner if it doesn't like it :)

Reply:
Done :)